src/components/layout.js

```diff
@@ -26,7 +26,7 @@ export default function Layout({ title, children }) {
         <Header />
         <Container maxWidth="lg" component="main">
           <Container maxWidth="sm">
-            <Typography variant="h1" className={classes.mainHeading}>
+            <Typography variant="h5" className={classes.mainHeading}>
```

Comment:
This doesn't seem related to this PR. Why did this change?

Reply:
I changed it because I thought it looked oversized compared to the new card/font sizes. I added component="h1" to it. Would you like me to move it to a different PR?